src/dev/docgenPlugin.ts

```diff
@@ -40,12 +47,19 @@ function docgenPlugin(): Plugin {
     name: "docgen-plugin",
     async configResolved(config) {
       config.logger.info(DOCGEN_LOG_MESSAGE);
-      await runDocGenProps();
+      const p1 = runDocGenProps();
+      const p2 = updateAllSnippets();
+      await Promise.all([p1, p2]);
```

Comment:
Do we really need two variables for that, as I don't see them reused somewhere else? Can't we just pass the functions to `Promise.all` directly?

Reply:
Yeah we could. I find this slightly more readable, but will change it.